mitosheet/src/components/taskpanes/AITransformation/AITransformationTaskpane.tsx

```diff
@@ -221,9 +251,31 @@ const AITransformationTaskpane = (props: AITransformationTaskpaneProps): JSX.Ele
                     <p className="text-color-error">
                         You do not have an OPEN_AI_KEY set in your enviornment variables. To activate this feature, follow the <a className='text-underline' href={DOCUMENTATION_LINK_AI_TRANSFORM} target='_blank' rel="noreferrer">instructions here.</a>
                     </p>
-                
                 }
-                <CollapsibleSection title={"Examples"} open={openSections['Examples']} disabled={apiKeyNotDefined}>
+                {(!apiKeyNotDefined && aiPrivacyPolicyNotAccepted) && 
+                    <>
+                        <p className="text-body-1">
+                            Mito AI is a beta feature. To improve the feature, we collect data used by the AI feature, including: dataframe names, column headers, and cell values.
+                        </p>
+                        <Spacer px={5}/>
+                        <p className="text-body-1">
+                            Behind the scenes, we use OpenAI to help generate code for Mito AI. As such, this data is also sent to OpenAI. You can see their <a className='text-underline' href='https://openai.com/policies/privacy-policy' target='_blank' rel="noreferrer">privacy policy here.</a>.
+                        </p>
+                        <Spacer px={5}/>
+                        <p className="text-body-1">
+                            You can see Mito's <a className='text-underline' href='https://privacy.trymito.io/privacy-policy' target='_blank' rel="noreferrer">privacy policy here.</a>
+                        </p>
+                        <TextButton
+                            onClick={() => {
+                                void props.mitoAPI.updateAcceptAITransformationPrivacyPolicy();
+                            }}
+                            variant='dark'
+                        >
+                            Accept Privacy Policy for Mito AI
```

Comment:
When they accept, can we log it? Seems like something that we'd want a record of.

Reply:
We do log it -- all update events are logged by default.